packages/connect-web/src/connectSettings.ts

```diff
@@ -71,6 +71,11 @@ export const parseConnectSettings = (input: Partial<ConnectSettings> = {}): Conn
         if (query['trust-issues'] === 'true') {
             settings.trustedHost = false;
         }
+
+        // Even if connect is running on not trusted domain we can override it with `trust-issues=false` query param.
+        if (query['trust-issues'] === 'false') {
+            settings.trustedHost = true;
+        }
```

Comment:
This is required if we want to support both `trust-issues` as well as setting page in connect-explorer.

In order to guarantee the current tests pass `trust-issues` is top in the hierarchy.

We can change it as we want later on and maybe use only settings page in connect-explorere and remove `trust-issues` from query param.

Reply:
does this work? it is not really supposed to work https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/data/connectSettings.ts#L72